src/Cache.php

```diff
@@ -30,20 +30,25 @@ class Cache
 
   protected $tokenArray;
 
+    /** @var bool $prettyPrint State of using JSON_PRETTY_PRINT for writing the JSON Files */
+    protected bool $prettyPrint = false;
+
   /**
    * Cache constructor.
    * @param string $storePath
    * @param array $cacheTokenArray
    * @param int|null $cacheLifetime
    */
-  public function __construct(string $storePath, array &$cacheTokenArray, $cacheLifetime)
+  public function __construct(string $storePath, array &$cacheTokenArray, $cacheLifetime, bool $prettyPrint)
```

Comment:
Type declarations are supported in PHP >= 7.4

Reply:
Done

Reply:
In PHP 7.0 we already have the support for boolen inside the method constructors. 

![grafik](https://user-images.githubusercontent.com/2758879/169668234-5538c897-d10a-4d53-ae82-519daadacc77.png)